dataframe-jdbc/src/test/kotlin/org/jetbrains/kotlinx/dataframe/io/mssqlTest.kt

```diff
@@ -0,0 +1,401 @@
+package org.jetbrains.kotlinx.dataframe.io
+
+import io.kotest.matchers.shouldBe
+import org.intellij.lang.annotations.Language
+import org.jetbrains.kotlinx.dataframe.DataFrame
+import org.jetbrains.kotlinx.dataframe.annotations.DataSchema
+import org.jetbrains.kotlinx.dataframe.api.*
+import org.jetbrains.kotlinx.dataframe.io.JdbcTest.Companion
+import org.jetbrains.kotlinx.dataframe.io.db.H2
+import org.junit.AfterClass
+import org.junit.BeforeClass
+import org.junit.Ignore
+import org.junit.Test
+import java.math.BigDecimal
+import java.sql.Connection
+import java.sql.DriverManager
+import java.sql.ResultSet
+import java.sql.SQLException
+import java.util.*
+import kotlin.reflect.typeOf
+
+private const val URL = "jdbc:sqlserver://localhost:1433;encrypt=true;trustServerCertificate=true"
+private const val USER_NAME = "root"
+private const val PASSWORD = "pass"
+private const val TEST_DATABASE_NAME = "testKDFdatabase"
+
+@DataSchema
+interface Table1MSSSQL {
+    val id: Int
+    val bigintColumn: Long
+    val binaryColumn: ByteArray
+    val bitColumn: Boolean
+    val charColumn: Char
+    val dateColumn: Date
+    val datetime3Column: java.sql.Timestamp
+    val datetime2Column: java.sql.Timestamp
+    val datetimeoffset2Column: String
+    val decimalColumn: BigDecimal
+    val floatColumn: Double
+    val imageColumn: ByteArray?
+    val intColumn: Int
+    val moneyColumn: BigDecimal
+    val ncharColumn: Char
+    val ntextColumn: String
+    val numericColumn: BigDecimal
+    val nvarcharColumn: String
+    val nvarcharMaxColumn: String
+    val realColumn: Float
+    val smalldatetimeColumn: java.sql.Timestamp
+    val smallintColumn: Int
+    val smallmoneyColumn: BigDecimal
+    val timeColumn: java.sql.Time
+    val timestampColumn: java.sql.Timestamp
+    val tinyintColumn: Int
+    val uniqueidentifierColumn: Char
+    val varbinaryColumn: ByteArray
+    val varbinaryMaxColumn: ByteArray
+    val varcharColumn: String
+    val varcharMaxColumn: String
+    val xmlColumn: String
+    val sqlvariantColumn: String
+    val geometryColumn: String
+    val geographyColumn: String
+}
+
+@Ignore
```

Comment:
Hmm yeah we really need to be able to test them locally/on TC. Maybe https://testcontainers.com/ can help?

Reply:
#668 is created for that also (will be fixed during this release cycle), testcontainers required preinstalled docker